packages/application-components/src/components/page-content-containers/page-content-wide/page-content-wide.tsx

```diff
@@ -0,0 +1,80 @@
+import { Children, ReactNode } from 'react';
+import styled from '@emotion/styled';
+
```

Comment:
All the new three components have a quite verbose implementation (many simple styles components and css naming that are not strictly necessary) but I found it to maybe be more clear to understand.
Also, I didn't move the style components to a `styles` sibling file.

Despite both things, I feel the components are small enough to keep everything in one file.